specs/order_spec.rb

```diff
@@ -80,31 +106,70 @@
 
 # TODO: change 'xdescribe' to 'describe' to run these tests
 xdescribe "Order Wave 2" do
+
+
   describe "Order.all" do
     it "Returns an array of all orders" do
       # TODO: Your test code here!
+      #arrange
+       # not neccessary for class methods
+      #act
+      result = Grocery::Order.all
+      #assert
+      #returns an array
+      result.must_be_kind_of Array
+      #returns the right number of elements
+      #more assertions checking other thigns
+      #each element is an instance and has right number of orders
     end
 
     it "Returns accurate information about the first order" do
       # TODO: Your test code here!
+      #act
+      first_order = CSV.open('support/orders.csv', 'r') { |csv| csv.first }
+      #assert
+      first_order == (Grocery::Order.all).first
```

Comment:
You don't have any assertions in this test, or any of the tests in this section! On line 131 you produce a boolean, but you don't do anything with it. If you want to assert equality, you need to say something like `first_order.must_equal (Grocer::Order.all).first`.

As is, as long as the code doesn't raise an error, this test passes vacuously - there's no way to make it fail.